tests/unit_tests/gui/plottery/test_plotting_of_snake_oil.py

```diff
@@ -33,9 +55,13 @@ def test_that_all_snake_oil_visualisations_can_be_shown(
         for i in range(key_list.model().rowCount()):
             key_list.setCurrentIndex(key_list.model().index(i, 0))
             selected_key = data_types.getSelectedItem()
-            for i, tab in enumerate(plot_window._plot_widgets):
-                if central_tab.isTabEnabled(i):
-                    central_tab.setCurrentWidget(tab)
-                    assert selected_key["dimensionality"] == tab._plotter.dimensionality
-                else:
```

Comment:
You lost this path, was that deliberate?

Reply:
Added it back :)